pride-core/src/main/java/com/prezi/pride/Pride.java

```diff
@@ -165,7 +165,16 @@ public void removeModule(final String name) throws IOException {
 		final File moduleDir = getModuleDirectory(name);
 		logger.info("Removing " + name + " from " + moduleDir);
 		modules.remove(name);
-		FileUtils.deleteDirectory(moduleDir);
+		try {
+			FileUtils.deleteDirectory(moduleDir);
+		}
+		catch (IOException ex) {
+			// Found on Windows only, sometimes files cannot be deleted
+			// This can be due to background processes monitoring the filesystem or
+			// the user left a command shell open in the folder
+			logger.error("Removing " + name + " failed with error: " + ex.getMessage());
+			logger.error("Please remove folder " + moduleDir + " manually");
```

Comment:
I agree that we should not stop at the first delete error. But shouldn't this change the exit value of the process as well?